tests/test_updater_with_simulator.py

```diff
@@ -115,7 +115,7 @@ def test_targets(self, test_case_data: Tuple[str, bytes, str]):
         updater = self._run_refresh()
         # target now exists, is not in cache yet
         info = updater.get_targetinfo(targetpath)
-        self.assertIsNotNone(info)
+        assert(isinstance(info, TargetFile))
```

Comment:
nitpick: I believe these type of asserts could still just check for none (`assert info is not None`) just in a form that mypy recognises.

Reply:
and why not `self.assertIsInstance` ?

Reply:
I tried  `self.assertIsInstance` and `mypy` doesn't recognize as it doesn't run our code, but just statically checks it.
However, what worked was what @jku suggested using `assert info is not None`.
Will change all places where I use `assert` with this approach.